runtime/tutor.txt

```diff
@@ -415,6 +422,303 @@ _________________________________________________________________
+=                         SETTING CASE                          =
+=================================================================
+
+ Press ` (above tab key on most keyboards) to set all selected
+ letters to lowercase.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. press x to select the line.
+ 3. press ` to change it to all lowercase
+
+--> this SENTENCE SHOULD all be in LOWERCASE!
+
+ Note: Conversely, alt-` changes all selected letter to uppercase.
+
+
+
+
+
+=================================================================
+=                            MACROS                             =
+=================================================================
+
+ Macros are just a way to record a set of actions that you want
+ to repeat.
+
+ Press Q to start recording a macro, you should see a poup at
+ the bottom of your screen. Press Q again to stop recording.
+ Press q to repeat your recorded macro.
+
+ 1. Move the cursor to the first line below marked -->.
+ 2. Press Q to start recording.
+ 3. Enter insert mode and type some text.
+ 4. Exit insert and press Q again to stop recording
+ 5. Move to the line below and press q to repeat the macro.
+
+ -->
+ -->
+
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Press y to yank (copy) text and p to paste
+   * Press space-Y and space-P to yank/paste on the system
+     clipboard
+
+ * Press ~ to alternate case of selected letters
+   * press ` and alt-` to set the case of selected layers to
+     upper and lower respectively
+
+ * Press Q to record and stop recording a macro
+   * Press q to repeat the recorded macro
+
+
+
+
+
+=================================================================
+=                      USING THE JUMPLIST                       =
+=================================================================
+
+ Helix can keep track of "jumps" which are big movements, like
+ jumping to the definition of a function in code. It stores
+ these in what's called the jumplist.
+
+ Press C-s (ctrl-s) to manually save your current position to
+ the jumplist.
+
+ Press C-i and C-o to move forward and backwards in the jumplist
+ respectively.
+
+ 1. Press C-s somewhere.
+ 2. Move far away in the file.
+ 3. Press C-o (just once!) to come back to where you saved.
+
+
+
+
+
+=================================================================
+=                      SEARCHING IN FILE                        =
+=================================================================
+
+ Press / to search forward in file, enter to confirm search.
+ Press n to go to the next search match.
+ Press N to go to the previous search match.
+
+ 1. Press / and type in a common word, like 'banana'.
+ 2. Press enter to confirm the search.
+ 3. press n and N to cycle through the matches.
+
+ Like select mode, searching also supports regex!
```

Comment:
There is no option to *not* use regex. Also, you mean the select *command*, not select mode. They are two different things.
```suggestion
 Like the select command, searching also uses regex.
```